packages/rollup-core/src/app/l1-to-l2-event-handling/block-batch-submitter.ts

```diff
@@ -11,8 +11,7 @@ const log: Logger = getLogger('block-batch-submitter')
 
 export class BlockBatchSubmitter implements BlockBatchListener {
   // params: [timestampHex, batchesArrayJSON, signedBatchesArrayJSON]
```

Comment:
I'm missing some context, but seems like this comment/method would need a `blockNumberHex` if my intuition serves me right.

Reply:
Ahh yeah, that's a relic. Gotta update that comment. Nice find!